clients/src/main/java/org/apache/kafka/clients/consumer/internals/AssignorSelection.java

```diff
@@ -79,6 +80,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return String.format("Assignor selection {type:%s, name:%s}", type, serverAssignor);
+        return "AssignorSelection(" +
+                "type=" + type +
+                ", serverAssignor='" + serverAssignor + '\'' +
```

Comment:
nit nit: I find it a better format to read the code if adding the separators at the end of the previous line for better alignment (having all the added `propId=` at the beginning of each line)

Reply:
This is a pretty common pattern to override toString. Do you mean making it doing?

```
AssignorSelection(
type=TYPE,
serverAssignor=assignor,
...);
```

Currently everything is in a single line. Though - this can be used for logging, so i wonder what would it look like it if we add line separator.

Reply:
Agree with the pattern, I was only referring to having the nit of having props aligned :
```
return "AssignorSelection(" +
                "**type**=" + type + **"," +**
                "**serverAssignor**='" + serverAssignor + '\''
                ...
```